pandas-stubs/core/frame.pyi

```diff
@@ -275,23 +275,23 @@ class DataFrame(NDFrame, OpsMixin):
     @overload
     def to_parquet(
         self,
-        path: FilePathOrBytesBuffer,
-        *,
-        engine: Literal["auto", "pyarrow", "fastparquet"] = ...,
-        compression: Literal["snappy", "gzip", "brotli"] = ...,
-        index: _bool | None = ...,
-        partition_cols: list | None = ...,
+        path: FilePath | WriteBuffer[bytes],
+        engine: ParquetEngine = ...,
+        compression: Literal["snappy", "gzip", "brotli"] | None = ...,
+        index: bool | None = ...,
+        partition_cols: list[HashableT] | None = ...,
+        storage_options: StorageOptions = ...,
         **kwargs,
```

Comment:
I see that you typed `**kwargs` as `Any` in `read_parquet()`, so why not do it here (or not type it there)?

Reply:
Added, thanks.